.github/workflows/build_and_test.yml

```diff
@@ -160,6 +160,20 @@ jobs:
       afterBuild: rustup target add wasm32-unknown-unknown && curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh && node ./scripts/normalize-version-bump.js && turbo run build-wasm -- --target nodejs --features tracing/release_max_level_info && git checkout . && mv packages/next-swc/crates/wasm/pkg packages/next-swc/crates/wasm/pkg-nodejs && node ./scripts/setup-wasm.mjs && NEXT_TEST_MODE=start TEST_WASM=true node run-tests.js test/integration/production/test/index.test.js test/e2e/streaming-ssr/index.test.ts
     secrets: inherit
 
+  test-unit:
+    name: test unit
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [1, 2, 3]
```

Comment:
I think these are fast enough we don't need to spin up three separate jobs for them